packages/my_at_localization/src/my_at_pose_node.py

```diff
@@ -137,6 +137,16 @@ def _broadcast_tf(
     def _broadcast_detected_tag(self, image_msg, tag_id, tag):
         pose = np.identity(4)
         pose[0:3, 0:3] = tag.pose_R
+        # inverse the rotation and tranformation comming out of the AT detector.
+        # The AP detector's output is the camera frame relative to the TAG
+        # According to: https://duckietown.slack.com/archives/C6ZHPV212/p1619876209086300?thread_ts=1619751267.084600&cid=C6ZHPV212
+        # While the transformation below needs TAG relative to camera
+        # Therefore we need to reverse it first.
+        pose = np.transpose(pose) # reverse orientation
+        pose_t = np.ones((4,1)) 
+        pose_t[:3,:] = tag.pose_t
+        pose_t = - np.matmul(pose,pose_t) # reverse translation
+        pose_t = pose_t[:3,:]
```

Comment:
How I inversed the transformation matrix: https://math.stackexchange.com/a/152686

Probably `np.linalg.inv` also would work.. Not tested.